app/api/api_v1/endpoints/assignment_router.py

```diff
@@ -41,4 +42,22 @@ def get_assignment_submissions(
         .filter_by(assignment_id=assignment_id) \
         .join(StudentModel) \
         .filter(StudentModel.student_onyen == onyen) \
-        .all()
\ No newline at end of file
+        .all()
+
+@router.post("/assignment/{assignment_id}/deadline", response_model=AssignmentSchema)
```

Comment:
This is more of a `put` than a `post`, because you're updating a resource within the database explicitly and not creating a new resource.

Reply:
You should actually be using `PATCH` since it's a partial update to the resource, use `PUT` when replacing an existing resource.

Reply:
@Hoid @frostyfan109 I changed to PATCH now in the recent commit